hosted/json-schemas/definitions.json

```diff
@@ -6,6 +6,10 @@
     "did": {
       "type": "string",
       "pattern": "^did:([a-z0-9]+):((?:(?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))*:)*((?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))+))((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(\/[^#?]*)?([?][^#]*)?(#.*)?$"
+    },
+    "decimalString": {
+      "type": "string",
+      "pattern": "^([0-9]+[.][0-9]+)$"
```

Comment:
so this requires that we always have a decimal, even for whole amounts? `14` would not suffice, it must be `14.0` or `14.000` etc (since there's no rule on # of decimal places after the `.`)?

Reply:
Yep that's right. We can change this, but in office hours last month we discussed wanting to keep two significant digits beyond the currencies minor unit, which would always result in a decimal number.